tasks/admin/components/init.js

```diff
@@ -38,6 +37,8 @@ const
 ;
 
 module.exports = function (callback) {
+    const github = require('../../config/admin/github'); // eslint-disable-line global-require
```

Comment:
Why was this necessary? If you leave the github const where it was before the eslint rule is already fulfilled, isn't it?
Now, you moved it here and have to disable the rule... i don't get the reason 🤔

Reply:
Try moving this require up, local build will fail, as the require tries to connect to GitHub. It is good practise in libs to only define things, but no execute them immediately. Can you rewrite the `config/admin/github.js` to a singleton pattern?

Reply:
Ok, understood. The whole github API integration seems to be broken anyway and using deprecated functions, so i approve your changes as is for now as it needs a refactoring separately and is not a task for local building but for repo-admin purposes only